<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherAppClass</class>
 <widget class="QMainWindow" name="WeatherAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeatherApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListWidget" name="intervalsList"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Filter by precipitation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="precipitationsEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Start hour</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="startEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="descEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="calculateBtn">
        <property name="text">
         <string>Calculate hours</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="weatherapp.qrc"/>
 </resources>
 <connections/>
</ui>
